scripts/metadata_parser.py

```diff
@@ -525,7 +525,7 @@ def parse_scheme_tables(filename):
                         line = lines[line_number]
                         words = line.split('|')
                         if len(words) == 1:
-                            if words[0] == '!!':
+                            if words[0].strip() == '!!':
```

Comment:
Wasn't that already done in a separate PR, or is this a different place?

Reply:
Yes, this fix was merged into master. I forgot to merge master changes back into this branch before submitting the PR. Fixed now.